Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlaygroundSettings.swift

```diff
@@ -0,0 +1,270 @@
+//  PaymentSheet Example
+//
+//  Created by David Estes on 5/31/23.
+//
+
+import Foundation
+
+struct PaymentSheetTestPlaygroundSettings: Codable, Equatable {
+    enum UIStyle: String, PickerEnum {
+        static var enumName: String { "UI" }
+
+        case paymentSheet
+        case flowController
+    }
+
+    enum Mode: String, PickerEnum {
+        static var enumName: String { "Mode" }
+
+        case payment
+        case paymentWithSetup = "payment_with_setup"
+        case setup
+
+        var displayName: String {
+            switch self {
+            case .payment:
+                return "Payment"
+            case .paymentWithSetup:
+                return "Pay+SFU"
+            case .setup:
+                return "Setup"
+            }
+        }
+    }
+
+    enum IntegrationType: String, PickerEnum {
+        static var enumName: String { "Type" }
+
+        // Normal: Normal client side confirmation non-deferred flow
+        case normal
+        /// Def CSC: Deferred client side confirmation
+        case deferred_csc
+        /// Def SSC: Deferred server side confirmation
+        case deferred_ssc
+        /// Def MC: Deferred server side confirmation with manual confirmation
+        case deferred_mc
+        /// Def MP: Deferred multiprocessor flow
+        case deferred_mp
+
+        var displayName: String {
+            switch self {
+            case .normal:
+                return "Client-side confirmation"
+            case .deferred_csc:
+                return "Deferred client side confirmation"
+            case .deferred_ssc:
+                return "Deferred server side confirmation"
+            case .deferred_mc:
+                return "Deferred server side confirmation with manual confirmation"
+            case .deferred_mp:
+                return "Deferred multiprocessor flow"
+            }
+        }
+    }
+
+    enum CustomerMode: String, PickerEnum {
+        static var enumName: String { "Customer" }
+
+        case guest
+        case new
+        case returning
+    }
+
+    enum Currency: String, PickerEnum {
+        static var enumName: String { "Currency" }
+
+        case usd
+        case eur
+        case aud
+        case gbp
+        case inr
+    }
+
+    enum MerchantCountry: String, PickerEnum {
+        static var enumName: String { "MerchantCountry" }
+
+        case US
+        case GB
+        case AU
+        case FR
+        case IN
+    }
+
+    enum APMSEnabled: String, PickerEnum {
+        static var enumName: String { "Automatic PMs" }
+
+        case on
+        case off
+    }
+
+    enum ShippingInfo: String, PickerEnum {
+        static var enumName: String { "Shipping info" }
+
+        case on
+        case onWithDefaults = "on w/ defaults"
+        case off
+    }
+
+    enum ApplePayEnabled: String, PickerEnum {
+        static var enumName: String { "Apple Pay" }
+
+        case on
+        case off
+        case onWDetails = "on w/details"
```

Comment:
this is a bit inconsistent with shipping info -- maybe change this to onWithDetails ?